### PR TITLE
WiP: improve documentation

### PR DESCRIPTION
**Pull request by puckey**
This is a work in progress pull request. Please do not merge yet as I will add commits as the following issue progresses: https://github.com/react-native-kit/react-native-track-player/issues/505

### CODE REVIEW

File: docs/API.md [hunk 1]
```diff
@@ -3,143 +3,134 @@ title: Getting Started
 permalink: /api/
 ---
 
-## Starting off
-First of all, you need to initialize the player.
+## Getting Started
+First of all, you need to set up the player. This usually takes less than a second:
```
Comment (biomancer) [inline docs/API.md]: I think originally it had meaning that the call itself is taking less than a second (and resolves instantly if it was already initialized), and after this change it looks more like 'it's easy to setup'

Comment (puckey) [reply docs/API.md]: Yes, the issue here for me was the difference between getting started and getting into implementation details.. would it make sense to move the instant resolving part to the setupPlayer function documentation?

**Comment by dcvz** [comment]
@puckey I just realised that it's labeled WIP. Should I merge it now, or did you mean to make more updates?

**Comment by puckey** [comment]
@dcvz Happy to help! I will give it a final run through later this week and will let you know when it is ready for merging.

**Comment by curiousdustin** [comment]
@puckey, this is from quite awhile ago. Have you had a chance to review it recently?

**Comment by puckey** [comment]
No - but I will do so - adding it to my todo list.

**Comment by puckey** [comment]
opening a new PR for this

**Comment by puckey** [comment]
@curiousdustin Feel free to merge this PR: https://github.com/react-native-kit/react-native-track-player/pull/925

**Pull request closed without merging**